pkg/cmd/scan/output/html_test.go

```diff
@@ -236,7 +236,7 @@ func TestHTML_Write(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			tempDir := t.TempDir()
-			tempFile, err := ioutil.TempFile(tempDir, "result")
+			tempFile, err := os.CreateTemp(tempDir, "result")
```

Comment:
Is this something related to this PR or just a refactoring ? Does not matter it's not a big deal, just want to be sure I'm not missing something here 🤔